independent-publisher-connectors/Airlabs/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "api_key",
+          "description": "The api_key for this api",
+          "tooltip": "Provide your api_key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@fordosa90 for all our independent publishers, we ask to set the iconBrandColor to "#da3b01". Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).

Reply:
correct catch, updating with new commit